sgt-back-end/index.js

```diff
@@ -0,0 +1,165 @@
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+const pg = require('pg');
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades/:gradeId', (req, res, next) => {
```

Comment:
Since you don't use `next` you should leave it off the parameter list. (Here and below)